docs/1. Setup Guide - Docker.md

```diff
@@ -190,8 +190,8 @@ You may sign up for an Azure account [here](https://azure.microsoft.com/en-us/),
 
 1. You do not have to do this if you've already setup S3!
 1. Log on to the [Azure Portal](https://portal.azure.com).
-1. In the left pane, click **Storage**.
-1. Select your storage account.
+1. From the Dashboard, click **Storage accounts** on the left.
+1. Click Add at the top of the page to create a new storage account. 
```

Comment:
Make both blob containers, and name one public and one private.
![image](https://cloud.githubusercontent.com/assets/13974084/24455797/e6743e88-145e-11e7-9835-0278b0a6f04f.png)